Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -1299,7 +1299,64 @@ public static int FindControls<TControl>(string path, ref InputControlList<TCont
 
         #region Events
 
-        public static event Action<InputEventPtr> onEvent
+        /// <summary>
+        /// Called during <see cref="InputSystem.Update"/> for each event that is processed.
+        /// </summary>
+        /// <remarks>
+        /// Every time the input system updates (see <see cref="InputSettings.updateMode"/>
+        /// or <see cref="InputSystem.Update"/> for details about when and how this happens),
+        /// it flushes all events from the internal event buffer that are due in the current
+        /// update (<see cref="InputSettings.timesliceEvents"/> for details about when events
+        /// may be postponed to a subsequent frame).
+        ///
+        /// As the input system reads events from the buffer one by one, it will trigger this
+        /// callback for each event before then proceeding to process the event. However, if
+        /// any of the callbacks sets <see cref="InputEvent.handled"/> to true, the event will
+        /// be skipped and ignored.
+        ///
+        /// Note that the input system does NOT sort events by timestamps (<see cref="InputEvent.time"/>).
+        /// Instead, they are consumed in the order they are produced. This means that they
+        /// will also surface on this callback in that order.
+        ///
+        /// Note that the callback will be called for any event picked up from the internal
+        /// event buffer. This may include events for devices that have not been created and thus
+        /// may mean that the <see cref="InputDevice"/> parameter to the callback is null.
+        ///
+        /// <example>
+        /// <code>
+        /// // Treat LMB+RMB as MMB.
```

Comment:
nitpick: LMB RMB MMB are not necessarily understood abbreviations

Reply:
Fixed

Reply:
where? :)

Reply:
Crap, turns out you actually have to push your changes for them to be visible :) Fixed for reals now.